examples/rollup/BUILD.bazel

```diff
@@ -22,7 +28,7 @@ load(":rollup.bzl", "rollup")
 rollup(
     name = "bundle",
     entry_point = "examples/rollup/foo.js",
-    srcs = ["foo.js", "bar.js"],
+    deps = ["foo.js", "bar.js", ":mock_ts_lib"],
```

Comment:
foo.js should still be in srcs, it's not the output of some other rule

Reply:
Donezo